pyramid_swagger/load_schema.py

```diff
@@ -88,6 +88,15 @@ def swagger_type_validator(validator, ref, instance, schema):
     return swagger_type_validator
 
 
+def type_validator(validator, types, instance, schema):
+    """Swagger 1.2 supports parameters of 'type': 'File'. Skip validation of
```

Comment:
If you don't special-case the 'File' validation, and instead just use the general-case validation, does it fail?